packages/edit-site/src/components/header-edit-mode/document-actions/index.js

```diff
@@ -126,14 +102,19 @@ export default function DocumentActions() {
 	}
 
 	// Return feedback that the template does not seem to exist.
-	if ( ! entityTitle ) {
+	if ( ! record ) {
 		return (
 			<div className="edit-site-document-actions">
 				{ __( 'Template not found' ) }
 			</div>
 		);
 	}
 
+	const entityLabel =
+		record.type === 'wp_template_part'
+			? __( 'template part' )
+			: __( 'template' );
```

Comment:
There was a bug in trunk where these strings were not translated.

Reply:
Couldn't we rely o the post type labels provided by the rest API?

Reply:
These labels are only available in plural form for all users. If we can reword here to use plural that would work, but let's leave this separate and just keep the bug fix for now.